mongo/integration/sessions_test.go

```diff
@@ -330,6 +330,23 @@ func TestSessions(t *testing.T) {
 	})
 }
 
+func TestSessionsStandalone(t *testing.T) {
+	standaloneOpts := mtest.NewOptions().MinServerVersion("3.6").Topologies(mtest.Single)
+	mt := mtest.New(t, standaloneOpts)
+	mt.RunOpts("doesn't send transaction number", standaloneOpts, func(mt *mtest.T) {
+		sess, err := mt.Client.StartSession()
+		assert.Nil(mt, err, "StartSession error: %v", err)
+		defer sess.EndSession(mtest.Background)
+
+		err = mongo.WithSession(mtest.Background, sess, func(ctx mongo.SessionContext) error {
+			doc := bson.D{{"foo", 1}}
+			_, err := mt.Client.Database("foo").Collection("bar").InsertOne(ctx, doc)
```

Comment:
I think this approach will leave a `foo.bar` collection remaining in the server once the test ends. Can we use `mt.Coll` here? If not, you should explicitly call `mt.CreateCollection` so the collection is tracked and dropped after the test.